raiden_contracts/contract_manager.py

```diff
@@ -17,12 +17,20 @@
 # Classes for static type checking of deployed_contracts dictionary.
 
 
-class DeployedContract(TypedDict):
-    address: Address
-    transaction_hash: str
-    block_number: int
-    gas_cost: int
-    constructor_arguments: Any
+DeployedContract = TypedDict(
```

Comment:
What's the advantage of `TypedDict` over a dataclass?

Reply:
I didn't compare them.  https://github.com/raiden-network/raiden-contracts/issues/974